### ISSUE TITLE
Document allowEmptyInput for Node API

### ISSUE THREAD

**Issue by vgurkov** [stylelint/stylelint#5866]
### Describe the documentation issue

Does the Node.js API have an equivalent for the `--allow-empty-input`  CLI argument?
We're working on a project where we're slowly updating an old codebase and most of the styles are ignored by default. 

### What solution would you like to see?

A way to allow empty input via the Node API in case all passed files are ignored in `.stylelintignore`

**Comment by doing-art** [comment]
Hello @vgurkov @jeddy3. `allowEmptyInput` seems to be already supported. It is just not documented. Link to [the code](https://github.com/stylelint/stylelint/blob/b37be75e014bdd14703671d688b48b9c286b9eb6/lib/standalone.js#L263-L264).

I have just checked it on my local machine and it works as expected:
### When `allowEmptyInput` is enabled (equal to `true`):
```js
const lint = require("../lib/standalone");

lint({
	config: {
		rules: {
			'rule-selector-property-disallowed-list': {
				'a': ['color', "/margin/"]
			}
		}
	},
	files: "**/*.css",
	formatter: "verbose",
	allowEmptyInput: true
})
	.then(function (result) {
		console.log("result", result);
	});
```
![изображение](https://user-images.githubusercontent.com/25134486/152697703-ca505140-fd7c-47c0-b103-a93d89ae2561.png)

### When `allowEmptyInput` is disabled (equal to `false`):
```js
const lint = require("../lib/standalone");

lint({
	config: {
		rules: {
			'rule-selector-property-disallowed-list': {
				'a': ['color', "/margin/"]
			}
		}
	},
	files: "**/*.css",
	formatter: "verbose",
	allowEmptyInput: true
})
	.then(function (result) {
		console.log("result", result);
	});
```
![изображение](https://user-images.githubusercontent.com/25134486/152697767-9c84449b-afb3-44c7-ab5d-b369b6c0b97b.png)

I created a tiny PR where documented `allowEmptyInput` option for the node js API.
https://github.com/stylelint/stylelint/pull/5883
